dds/DCPS/Service_Participant.cpp

```diff
@@ -1557,6 +1557,14 @@ Service_Participant::load_configuration(
                      -1);
   }
 
+  if (status != 0) {
+    ACE_ERROR_RETURN((LM_ERROR,
+    ACE_TEXT("(%P|%t) ERROR: Service_Participant::load_configuration ")
+    ACE_TEXT("load_common_configuration () returned %d\n"),
+    status),
+    -1);
```

Comment:
Indent is missing and this should probably be a conditional notice log with a separate return.